### PR TITLE
Update readme.md

### LINKED ISSUE #3: Updating Readme
- [x] a summary paragraph that describes the project at a high level @tdkhanhvu 

- [x] a bulleted list of the functions (and datasets if applicable) that will be included in the package (this should be a 1-2 sentence description for each function/dataset) @yhchen20 

- [x] a paragraph describing where your packages fit into the Python ecosystem (are there any other Python packages that have the same/similar functionality? Provide links to any that do. If none exist, then clearly state this as well). @yhchen20 and @jianructose 

- [x] add contributors name @yhchen20

### PR DESCRIPTION
**Pull request by yhchen20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,22 @@
 ## Package Overview
 `pygtracker` is a Python package that allows UBC lecturers to record, analyze and adjust grades for students in a particular program. Users can record grades from each course, generate a summary report to determine which class is more challenging than the rest, or identify students who may need help. Finally, the package can suggest ways to adjust the grades for students to ensure the average grade matches a predefined benchmark. It contains five functions: `register_courses`, `record_grades`, `summarize_courses`, `rank_students` and `suggest_grade_adjustment`.
 
+The main components of this package are:
+
+- Register the courses
+  - Read/store the courses information as a dataframe
+- Record the grade for students
+  - Read/store the students grade for each assessment as a data frame
```
Comment (tdkhanhvu) [inline README.md:13]:
```suggestion
  - Read/store the students' grades for each assessment as a data frame
```

File: README.md [hunk 2]
```diff
@@ -5,6 +5,22 @@
...
+- Summarize the grades by course
```
Comment (tdkhanhvu) [inline README.md:14]:
```suggestion
- Summarize grades by courses
```

File: README.md [hunk 3]
```diff
@@ -5,6 +5,22 @@
 ## Package Overview
 `pygtracker` is a Python package that allows UBC lecturers to record, analyze and adjust grades for students in a particular program. Users can record grades from each course, generate a summary report to determine which class is more challenging than the rest, or identify students who may need help. Finally, the package can suggest ways to adjust the grades for students to ensure the average grade matches a predefined benchmark. It contains five functions: `register_courses`, `record_grades`, `summarize_courses`, `rank_students` and `suggest_grade_adjustment`.
 
+The main components of this package are:
+
+- Register the courses
```
Comment (tdkhanhvu) [inline README.md:10]:
```suggestion
- Register courses
```

File: README.md [hunk 4]
```diff
@@ -5,6 +5,22 @@
...
+  - Read/store the courses information as a dataframe
+- Record the grade for students
```
Comment (tdkhanhvu) [inline README.md:12]:
```suggestion
- Record grades for students
```

File: README.md [hunk 5]
```diff
@@ -5,6 +5,22 @@
...
+  - Read/store the students grade for each assessment as a data frame
+- Summarize the grades by course
+  - Provide summary statistics on the courses 
+  - Provide the ranking of courses based on courses' average grade
+- Summarise the grades by students
```
Comment (tdkhanhvu) [inline README.md:17]:
```suggestion
- Summarise grades by students
```

File: README.md [hunk 6]
```diff
@@ -5,6 +5,22 @@
...
+  - Provide the ranking of students' average grades for the course selected (or the whole program).
+- Suggest grade adjustment
+  - Suggest grade adjustments for any course based on a predefined benchmark
```
Comment (tdkhanhvu) [inline README.md:20]:
```suggestion
  - Suggest grade adjustments for any course based on predefined benchmarks
```
(We have 3 benchmarks: course, lab, quiz)

**Review by tdkhanhvu — COMMENTED** [review]
The overall content is good. I only have some suggestions with regard to the "modifier" in grammar, not sure if they are correct or not

**Review by jianructose — APPROVED** [review]
good

**Pull request merged**